### DISCUSSION TITLE
MAP Questions

### DISCUSSION THREAD

**Discussion by AseKarlsson** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#570] [Q&A] [answered]
Hi All,

What a great vacuum cleaner S7 is and what an awesome integration from Piotr to HA.
Just today received my unit and integrated it into HA.

2 questions I hope to get some clarification on:

1.	Why does the rest of the house turn out orange, but one room still blue?
 
![MAP](https://user-images.githubusercontent.com/80900477/231543487-931d0e68-a6bc-449e-b4c4-5d988dbd3825.jpg)

2.	Can I map rooms (setup rooms, setup borders and room names) in HA or do I need to use the MiHome App and can that info come over to HA?

Thanks,
AseKarlsson

**Comment by PiotrMachowski** [comment, marked as answer]
1. It depends on your config
2. https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/317

**Reply by AseKarlsson** [reply]
Thanks Piotr!
